plugins/GSdx/resource.h

```diff
@@ -121,6 +121,7 @@
 #define IDC_OSDBUTTON                   2117
 #define IDC_OSD_MAX_LOG                 2118
 #define IDC_OSD_MAX_LOG_EDIT            2119
+#define IDC_OFFSETHACK_TEXT             2120
```

Comment:
I know, it should be troublesome to increment the value of all the macro constants, but I feel it's worth it. ;)